mantidproject/mantid
# VESUVIO l1 and theta resolution routine

Fixes #12671 
To test:
- Open the algorithm
- Select the `IP0005.dat` file form unit test data
- Provide names for the distribution workpsaces
- Run
- See that both distributions look Gaussian like

Release notes change is [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&action=historysubmit&diff=24173&oldid=24165).
